### PR TITLE
Add placeholder for self-hosting quick start, and add development quickstart guide

### PR DESCRIPTION
**Pull request by txus**
Just enough to get started hacking!

### CODE REVIEW

**Comment by txus** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by txus** [comment]
recheck

File: docs/guides/self-hosted/development.mdx
Comment (andresgutgon) [inline docs/guides/self-hosted/development.mdx:10]: We use https://orbstack.dev/ and is way superior. Can you link here as an alternative please?

**Review by andresgutgon — APPROVED** [review]
Thanks Txus!

File: docs/guides/self-hosted/development.mdx
Comment (andresgutgon) [reply docs/guides/self-hosted/development.mdx:10]: We should promote tech that works great. We all use it because is way better than standard Docker client

Comment (geclos) [reply docs/guides/self-hosted/development.mdx:10]: let's not debate this please

File: docs/guides/self-hosted/quick-start.mdx
Comment (geclos) [inline docs/guides/self-hosted/quick-start.mdx:6]: let's remove this page for now unless we fill it with content

File: docs/guides/self-hosted/development.mdx
Comment (geclos) [inline docs/guides/self-hosted/development.mdx:37]: @samulatitude not sure if we need to add this to the sidebar config or something

File: docs/guides/self-hosted/development.mdx
Comment (andresgutgon) [reply docs/guides/self-hosted/development.mdx:10]: I'll leave it to txus's judgment

Comment (txus) [reply docs/guides/self-hosted/development.mdx:10]: Since I mentioned the compatible alternative, I think that's ok, it's left for advanced users (I use podman & podman compose). In the end, tmuxinator calls `docker compose`, so it needs to be anything compatible with that interface, doesn't matter too much what it is.

File: docs/guides/self-hosted/quick-start.mdx
Comment (txus) [reply docs/guides/self-hosted/quick-start.mdx:6]: The issue though is that there is currently a link to this page from the main quick start guide, which confusingly redirects back to the getting started introduction.
image

Comment (geclos) [reply docs/guides/self-hosted/quick-start.mdx:6]: lol ok let's move forward then cc @samulatitude this probably needs reviewing at some point

**Review by geclos — DISMISSED** [review]
thanks 🙏🏼 ! will merge once conflict with main is solved

**Comment by txus** [comment]
@geclos i think it's fixed~

**Pull request merged**
